about.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>YayCommerce's About Page</title>
+    <script src="https://use.fontawesome.com/75d401af0c.js"></script>
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Barlow+Semi+Condensed:ital,wght@0,200;0,300;0,400;0,700;1,300&family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,300;0,400;0,600;1,200&family=Sora:wght@100;200;300;400;500;600;700;800&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <div class="social-login">
+        <div class="container">
+          <ul class="socila-login-items">
+            <li>
+              <a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li><a href="#">Logout</a></li>
+            <li><a href="#">Mypage</a></li>
+            <li><a href="#">Amharic</a></li>
+          </ul>
+        </div>
+      </div>
+      <nav class="nav-bar">
+        <div class="container">
+          <div class="logo">
+            <a href="index.html"
+              ><img
+                src="assets/images/yaycommerceNobg.png"
+                alt="logo"
+                class="logo"
+            /></a>
+          </div>
+          <ul class="navigation">
+            <li><a href="about.html" class="home">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#" class="contact">Buy Now</a></li>
```

Comment:
About page should have the save navbar as in the home page to look more consistent. As you can see, On the about page the home page nav item is missing. Although you've put the home page link in the logo to look more consistent the navbar should look the same. 😄